docs/installation.rst

```diff
@@ -1,15 +1,17 @@
 Installation
 ============
 
+.. contents::
+   :local:
+
 Requirements
 ************
 
-- In order to build images, you need to have either ``podman`` or ``docker``
-  installed as well as the ``ansible-builder`` Python package.
-  The ``--container-runtime`` option needs to correspond to the Podman/Docker
-  executable you intend to use.
-- ``ansible-builder`` version ``1.1.0`` requires Python ``3.8`` or higher.
-
+- To build images, you must install either ``podman`` or ``docker``
+  as well as the ``ansible-builder`` Python package.
+- The ``--container-runtime`` option needs to correspond to the Podman/Docker
+  executable you intend to use (where is this set?).
```

Comment:
@Akasurde @nitzmahone can you answer this question ^^ ?

Reply:
I think I understand this now. I'll update the text in my next changeset.